openhands_cli/acp_impl/main.py

```diff
@@ -0,0 +1,26 @@
+"""OpenHands ACP Main Entry Point."""
+
+import asyncio
+import logging
+import sys
+
+from .agent import run_acp_server
+
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    handlers=[logging.StreamHandler(sys.stderr)],
+)
+
+logger = logging.getLogger(__name__)
+
+
+async def run_acp_agent() -> None:
```

Comment:
NIT: do we need this function wrap, or can we use `run_acp_server` interchangeably ?